dashboard/app/models/plc/course.rb

```diff
@@ -10,5 +10,5 @@
 
 class Plc::Course < ActiveRecord::Base
   has_many :plc_enrollments, class_name: '::Plc::UserCourseEnrollment', foreign_key: 'plc_course_id', dependent: :destroy
```

Comment:
Could you add a README.md in this directory that links to your design doc and includes the relationships diagram?